petshop/src/main/java/ua/hillel/petshop/config/AppConfig.java

```diff
@@ -0,0 +1,55 @@
+package ua.hillel.petshop.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+
+@Configuration
+public class AppConfig {
+
+    Clock clock = Clock.system(ZoneId.of("Europe/Kiev"));
+
+    Instant kharkivTime = Instant.now().plus(2L, ChronoUnit.HOURS);
+
+    private final String appName = "Pet Shop";
+
+    private final String appAuthor = "Ihor Sutulov";
+
+    public Clock getClock() {
+        return clock;
+    }
+
+    public Instant getKharkivTime() {
+        return kharkivTime;
+    }
+
+    public String getAppName() {
+        return appName;
+    }
+
+    public String getAppAuthor() {
+        return appAuthor;
+    }
```

Comment:
I don't see a big value of using using getters in this class. it is better to remove it. and use class variables in bean